solhint.js

```diff
@@ -128,12 +132,23 @@ const readConfig = _.memoize(() => {
   let config = {}
 
   try {
-    const configStr = fs.readFileSync('.solhint.json').toString()
+    const configFile = program.config || '.solhint.json'
+
+    if (!fs.existsSync(configFile)) {
+      throw new FileNotExistsError('The config file doesnt exist')
+    }
+
+    const configStr = fs.readFileSync(configFile).toString()
+
     config = JSON.parse(configStr)
   } catch (e) {
     if (e instanceof SyntaxError) {
       console.log('ERROR: Configuration file [.solhint.json] is not a valid JSON!\n')
-      process.exit(0)
+      process.exit(1)
+    }
+    if (e instanceof FileNotExistsError) {
+      console.log(`ERROR: Configuration file [${program.config}] doesn't exist!\n`)
+      process.exit(1)
```

Comment:
Can't this two `process.exit(1)` lines be simplified in just one after both `if` statements?

Reply:
Done